aisynphys/connectivity.py

```diff
@@ -160,12 +164,22 @@ def measure_connectivity(pair_groups, alpha=0.05, sigma=None):
             'probed_pairs': probed_pairs,
         }
 
-        if sigma is not None:
+        if sigma is not None or fit_model is not None:
             distances = np.array([p.distance for p in probed_pairs], dtype=float)
             connections = np.array([p.synapse for p in probed_pairs], dtype=bool)
             mask = np.isfinite(distances) & np.isfinite(connections)
+            results[(pre_class, post_class)]['probed_distances'] = distances[mask]
+            results[(pre_class, post_class)]['connected_distances'] = connections[mask]
+        if sigma is not None:
             adj_conn_prob, adj_lower_ci, adj_upper_ci = distance_adjusted_connectivity(distances[mask], connections[mask], sigma=sigma, alpha=alpha)
             results[(pre_class, post_class)]['adjusted_connectivity'] = (adj_conn_prob, adj_lower_ci, adj_upper_ci)
+        if fit_model is not None:
+            if sigma is not None:
+                fit = fit_model.fit(distances[mask], connections[mask], method='L-BFGS-B', fixed_size=sigma)
+            else:
+                fit = fit_model.fit(distances[mask], connections[mask], method='L-BFGS-B')
```

Comment:
I think you can replace this with just:

    fit = fit_model.fit(distances[mask], connections[mask], method='L-BFGS-B', fixed_size=sigma)